Documentation/Contributors/BuildGuide/README.md

```diff
@@ -1,10 +1,34 @@
 # Build Guide
 
+- [Quickstart](#quickstart)
 - [Get the Code](#get-the-code)
 - [Build the Code](#build-the-code)
 - [Build Scripts](#build-scripts)
 - [Travis and Continuous Integration](#travis-and-continuous-integration)
 
+## Quickstart
+
+1. [Clone the Reop](#get-the-code)
+
+2. [Build](#build-the-code)
+
+   ```
+    npm install \
```

Comment:
```suggestion
    npm install
```

Reply:
I recommend leaving the escape, because if someone copy / pastes, it will prevent the terminal from running both commands at the same time.

Reply:
Ah, I see your point. We don't use that anywhere else in our documentation though ([for example](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodeReviewGuide#rebase)), so just for the sake of being constant, let's remove.